CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ find_package(rmw_implementation_cmake REQUIRED)
 find_package(std_msgs REQUIRED)
 
 # find ROS 1 packages
+list(APPEND ros1_cmake_macros cmake/find_ros1_package.cmake cmake/find_ros1_interface_packages.cmake)
```

Comment:
Since the variable `ros1_cmake_macros` doesn't exist yet use `set()` instead. Based on the usage I would also suggest to name the variable `cmake_extra_files`.